CONTRIBUTING.md

```diff
@@ -0,0 +1,23 @@
+## Releasing a new npm version
```

Comment:
It is also possible to use automatic releasing

Reply:
I'm not looking to build an automatic release system at this time or in this PR.